packages/decoder/lib/decoders.ts

```diff
@@ -126,7 +126,7 @@ export class WireDecoder {
     ({
       definitions: this.referenceDeclarations,
       types: this.userDefinedTypes
-    } = this.collectUserDefinedTypes());
+    } = this.collectUserDefinedTypesEventsAndErrors());
```

Comment:
I mean, it now collects events and errors as well, if I didn't rename it it would be unclear.  I'd prefer to rename it so the name continues to be descriptive.  This method isn't part of the public interface or anything so this isn't breaking or anything.  (Also it's going away in `wrap`, or will be once I merge this into that and make the appropriate changes.)

Reply:
(I suppose the `EventsAndErrors` part of the name could change per below...)

Reply:
Sure sounds good. The `EventsAndErrors` part is what bothered me :)